.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,13 @@
 #### Description
 
-#### Screenshot (if UI related)
+#### Screenshot (if UI-related)
 
-#### Todos
+#### To-Dos
 
-- [ ] Sucessfully builds `yarn build`
-- [ ] Translation Keys `yarn i18n:extract`
-- [ ] Database migration created (if required)
+- [ ] Successful build `yarn build`
+- [ ] Translation keys `yarn i18n:extract`
+- [ ] Database migration (if required)
 
-#### Issues Fixed or Closed by this PR
+#### Issues Fixed
```

Comment:
What's wrong with "or closed by this PR"? Not all PR's are fixes. Some are feature requests which close issues, not fix them.

Reply:
Ah true.  I do think the "by this PR" part is unnecessary though haha.